debezium-quarkus-outbox/pom.xml

```diff
@@ -23,6 +23,12 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-testing-testcontainers</artifactId>
+                <version>${project.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
I'm sorry, we meant the parent-parent-pom, in the root dir.

Reply:
Uh my bad, will change it. :sweat_smile: